tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -0,0 +1,157 @@
+/***************************************************************************
+*  Copyright (C) Codeplay Software Limited
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  For your convenience, a copy of the License has been included in this
+*  repository.
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+**************************************************************************/
+
+#ifndef ONEMKL_COMPUTE_INPLACE_HPP
+#define ONEMKL_COMPUTE_INPLACE_HPP
+
```

Comment:
I've not commented on this, it's a weird design because these files are included at the end of compute_tester.hpp. I was thinking we could change it something more conventional but I don't mind too much.

Reply:
Initially I had everything on compute_tester.hpp but the file was starting to be huge. And there will be more functions added to it in the future. So was trying to split the code a bit by doing this. 

Open to suggestions though. Can also rename the files to some different extention to make clear they are not regular hpp files.

Reply:
A more common approach would be to:

- Remove the current includes at the end of compute_tester.hpp
- Include compute_tester.hpp at the top of compute_inplace.hpp and similar files
- Include compute_inplace.hpp and similar in compute_tests.cpp

Reply:
Sure, if that's more conventional I can do that